include/odp/api/spec/ipsec.h

```diff
@@ -1346,9 +1346,7 @@ int odp_ipsec_in(const odp_packet_t pkt_in[], int num_in,
  * and content of packet data before the IP header is undefined. Use outbound
  * operation parameters to specify the amount of TFC padding appended to
  * the packet during IPSEC transformation. Options can be used also to create
- * TFC dummy packets. Packet data content is ignored in tunnel mode TFC dummy
- * packet creation as tfc_pad_len option defines solely the packet length.
- * In all other cases, payload length for the IPSEC transformation is specified
+ * TFC dummy packets. Payload length for the IPSEC transformation is specified
  * by odp_packet_len() minus odp_packet_l3_offset() plus tfc_pad_len option.
```

Comment:
Unless it has been parsed, `odp_packet_l3_offset()` is initialized to `ODP_PACKET_OFFSET_INVALID`, so this seems an undue burden. The original wording seems cleaner from an application perspective.

Reply:
We require that an application sets `l3_offset` for all packet it pushes to IPsec. For TFC dummy packets it resulted in `l3_offset` being set but ignored. Thus I proposed this change. Other solution might be to stop requiring `l3_offset` for TFC dummy packets.

Reply:
A dummy packet is defined only by the fact that it is a dummy and the requested length. No further metadata should be needed or required, so I'd be happy to see that requirement dropped. I view this as a building block for a TBD higher-level configuration service that would simply permit IPsec to be configured to provide a requested level of traffic masking and leave it up to the implementation to decide how best and most efficiently to achieve that via a combination of TFC dummy packets and padding.